### ISSUE TITLE
Request: funnel icon

### ISSUE THREAD

**Issue by atrakh** [radix-ui/icons#165]
Would be great to represent filters

**Comment by jkeussen** [comment]
There are already two good icons that can work for filtering: `MixerHorizontal` and `MixerVertical`.

However, I took a pass at a generic funnel/filter icon. What do you think of these?

![image](https://github.com/radix-ui/icons/assets/1137557/c084d9f3-2650-4304-af2a-e3474922c7ce)
![image](https://github.com/radix-ui/icons/assets/1137557/24d6834f-f547-40a7-b0c6-f46e69871f96)
<img width="289" alt="Screenshot 2023-08-24 at 01 32 33" src="https://github.com/radix-ui/icons/assets/1137557/979c6fc7-8fb5-40a1-9085-b3dbc0d5adf0">

**Comment by atrakh** [comment]
They both look great! I’d used the non-filled variant

I use Mixer for settings/config so its nice to distinguish filters from that

Thank you

**Comment by lkpnchl** [comment]
I agree, having this icon will be great for filters. I hope radix-ui will have one soon too.
